From 448f096e5c3a139535f162dfbfe8c08c434ac742 Mon Sep 17 00:00:00 2001
From: "Lee, Chun-Yi" <jlee@suse.com>
Date: Wed, 15 Dec 2021 19:44:58 +0800
Subject: [PATCH] MokManager: removed Locate graphic output protocol fail
 error message

On some platform, like GCP or OVMF which does NOT provide
EFI_GRAPHICS_OUTPUT_PROTOCOL when no display device (or the display device
be disabled). It causes that the "Error: Locate graphic output protocol
fail: (0xE) Not Found." always be showed on console when we enroll MOK
through MokManager. The message box blocked the process of enrolling
MOK and scared user. The error message is introduced by 55163bc82c517 since
15.2. This patch removed the error message.

Signed-off-by: Lee, Chun-Yi <jlee@suse.com>
---
 lib/console.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/console.c b/lib/console.c
index 7be5d54..cca9f0a 100644
--- a/lib/console.c
+++ b/lib/console.c
@@ -522,7 +522,6 @@ console_mode_handle(VOID)
 
 	efi_status = BS->LocateProtocol(&gop_guid, NULL, (void **)&gop);
 	if (EFI_ERROR(efi_status)) {
-		console_error(L"Locate graphic output protocol fail", efi_status);
 		return;
 	}
 
-- 
2.25.1

